packages/react/src/components/Provider/Provider.tsx

```diff
@@ -181,6 +184,7 @@ class Provider extends React.Component<WithAsProp<ProviderProps>> {
       theme,
       rtl,
       disableAnimations,
+      enableCaching,
```

Comment:
Maybe the value should be `disableCaching` as we already have `disableAnimations` in the context.. But it feels wrong, as I want to disable it by default...